src/manual/coupon-decorators.md

```diff
@@ -28,37 +37,43 @@ Tests if the total value of the basket (including service costs for delivery and
 
 basketvalues.total-value-min (optional)
 : The minimum value before the coupon has any effect. The format is a JSON encoded map of the currency ID and the value in x.xx format, e.g. *{"EUR":"1.00","USD":"1.00"}*
+To easliy enter the desired values, click the "+" icon to open a convenient input form:
```

Comment:
Think, there should be a newline before and after this line. At least before, so it isn't part of the definition before.